### ISSUE TITLE
error when running genReads.py

### ISSUE THREAD

**Issue by michalsharabi** [zstephens/neat-genreads#75]
hi
when running 
genReads.py -r GCA_000001405.21_GRCh38.p6_genomic.fna -R 126 -o sim_tst --bam --vcf --pe 300 30

I'm getting this error:
Using default sequencing error model.
Traceback (most recent call last):
  File "/home/users/bioinf/michalsha/work/unity/python/tools/simulating_ngs_data/genReads.py", line 179, in <module>
    SE_CLASS = ReadContainer(READLEN, SE_MODEL, SE_RATE)
  File "/home/users/bioinf/michalsha/work/unity/python/tools/simulating_ngs_data/py/SequenceContainer.py", line 662, in __init__
    errorDat = pickle.load(open(errorModel,'rb'))
UnicodeDecodeError: 'ascii' codec can't decode byte 0xb0 in position 0: ordinal not in range(128)

can you please help me,how do I resolve this issue
Thanks,
Michal

**Comment by zstephens** [comment]
Greetings,

This error looks similar to one I've encountered before, involving pickle files that were created in python 2 causing issues if you tried to read them in in python 3, and vice versa. Is it possible that somehow the default sequencing error model (errorModel_toy.p) got replaced with one that was created in python 3?

I'd recommend cloning a fresh copy of the latest repo, and seeing if the error persists.

-Zach
